### PR TITLE
Fix available layers stats when VLA is used with a single/default layer

### PR DESCRIPTION
**Pull request by murillo128**
If VLA is used with a single layer stream with no layer info (for example an h264 stream), a new `LayerSource` was created for storing the target bitrate/width/heigh/fps, but it was not used for calculating the other stats

So when the stats where gathered, as there was already a single layer, the code to create a default layer for the source if no layer info was present, was not run, and therefore the bitrate field was not exposed correctly.

With the change on the media server, we are adding the target stats to the `RTPIncomingSource` as well and not creating a `LayerSource`.

This PR exposes the target stats at source level and so the `getActiveLayers()` provide the correct info.

### CODE REVIEW

File: src/RTPIncomingSource.i [hunk 1]
```diff
@@ -40,5 +40,15 @@ struct RTPIncomingSource : public RTPSource
...
+		const int64_t targetFps;
 	}
-};
\ No newline at end of file
+};
+%{
+int64_t RTPIncomingSource_targetBitrate_get(RTPIncomingSource* self)	{ return self->value_or(0); } 
+int64_t RTPIncomingSource_targetWidth_get(RTPIncomingSource* self)	{ return self->value_or(0); }  
+int64_t RTPIncomingSource_targetHeight_get(RTPIncomingSource* self)	{ return self->value_or(0); } 
+int64_t RTPIncomingSource_targetFps_get(RTPIncomingSource* self)	{ return self->value_or(0); }
```
Comment (mildsunrise) [inline src/RTPIncomingSource.i:53]: is this correct?

Comment (mildsunrise) [reply src/RTPIncomingSource.i:53]: shouldn't it be `self->targetBitrate.value_or(0);` rather than `self->value_or(0);`?

**Review by mildsunrise — APPROVED** [review]

File: src/RTPIncomingSource.i (hunk 1 above)
Comment (murillo128) [reply src/RTPIncomingSource.i]: yes

Comment (murillo128) [reply src/RTPIncomingSource.i]: should be solved

**Pull request merged**